src/workerd/api/queue.c++

```diff
@@ -523,6 +523,12 @@ kj::Promise<WorkerInterface::CustomEvent::Result> QueueCustomEventImpl::run(
   // reuse the scheduled worker logic and timeout for now.
   auto completed = co_await incomingRequest->finishScheduled();
 
+  // Log some debug info if the queue handler completed but other waituntils did not
+  if (!completed && queueEventHolder->event->getIsQueueHandlerComplete()) {
+    auto taskTrace = incomingRequest->getContext().getWaitUntilTasks().trace();
+    KJ_LOG(WARNING, kj::str("Queue event timed out with lingering waitUntil tasks:\n", taskTrace));
```

Comment:
For the sake of the runtime team, we should probably go ahead and NOSENTRY this despite what I said earlier (i.e. add the text "NOSENTRY" at the beginning of the log message). We're just as capable of getting the info we need from logs as we are from sentry events, and that's less disruptive to folks who watch sentry for new problems.

Reply:
Added the NOSENTRY